integration-test/src/main/java/uk/gov/gchq/gaffer/integration/impl/ImportExportIT.java

```diff
@@ -0,0 +1,145 @@
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ImportExportIT extends AbstractStoreIT {
+
+    private static final String INCOMING_FILE_PATH = "NeptuneEntitiesAndEdgesWithProperties.csv";
+    private Path path;
+    private File outgoingFile;
+    @TempDir
+    Path tempDir;
+
+    @Override
+    public void _setup() throws Exception {
+
+        try {
+            path = tempDir.resolve("outputFile.csv");
+        } catch (final InvalidPathException ipe) {
+            throw new InvalidPathException(ipe.getInput(), "error creating temporary test file in " + this.getClass().getSimpleName());
+        }
+
+        outgoingFile = path.toFile();
+    }
+
+    @Test
+    public void shouldImportFromFileThenExportBackOutToDifferentFile() throws OperationException, IOException {
+        // Given
+        List<String> expectedData = Arrays.asList(
+                ":ID,:LABEL,:TYPE,:START_ID,:END_ID,count:Int,DIRECTED:Boolean",
+                "A,BasicEntity,,,,1,",
+                "B,BasicEntity,,,,2,",
+                ",,BasicEdge,A,B,1,true"
+        );
+
+
+        final OperationChain<Iterable<? extends String>> importExportOpChain = new OperationChain.Builder()
+                .first(new ImportFromLocalFile.Builder()
+                        .filePath(INCOMING_FILE_PATH)
+                        .build())
+                .then(new CsvToElements.Builder()
+                        .csvFormat(new NeptuneFormat())
+                        .build())
+                .then(new AddElements.Builder()
+                        .build())
+                .then(new GetAllElements())
+                .then(new ToCsv.Builder()
+                        .csvFormat(new NeptuneFormat())
+                        .build())
+                .then(new ExportToLocalFile.Builder()
+                        .filePath(outgoingFile.getAbsolutePath())
+                        .build())
+                .build();
+
+        final Iterable<? extends String> exportedData = graph.execute(importExportOpChain, getUser());
+
+        //   When
+
+        Iterable<String> incomingFile = readFile(INCOMING_FILE_PATH);
+        Iterable<String> outgoingFile = readFile(path.toFile().getAbsolutePath());
+
+
+        //  Then
+        assertThat(outgoingFile).containsExactlyInAnyOrderElementsOf(incomingFile);
+    }
+
+
+    public Iterable<String> readFile(final String filePath) throws OperationException {
+        final OperationChain<Iterable<String>> importOpChain = new OperationChain.Builder()
+                .first(new ImportFromLocalFile.Builder()
+                        .filePath(INCOMING_FILE_PATH)
+                        .build())
+                .build();
+
+        return graph.execute(importOpChain, getUser());
+    }
+    protected Schema createSchema() {
```

Comment:
I'd add an `@Override` to make it clearer that it is not just a random method with no calls